src/artemis/tests/test_main_system.py

```diff
@@ -38,6 +38,9 @@ def abort(self):
                 raise Exception(self.error)
         self.RE_takes_time = False
 
+    def subscribe(self, *args: Any):
```

Comment:
Should: Can you add a test somehwere in this file that an FGSCommunicator gets passed into this at some point